worker/games.py

```diff
@@ -1323,11 +1319,34 @@ def parse_options(s):
                     sep="",
                     file=sys.stderr,
                 )
+
+    def create_environment():
+        white_list = ["COMPCXX"]
```

Comment:
I was joking in keeping `COMPCXX`, it's the dangerous one. :(
Perhaps it's better to keep the white_list for a while in case of issue.

Reply:
Ah ok. Do you want to keep the white list as it is now. Or just use 
```
white_list = []
```

Reply:
What I write in the OP would be an enhancement on top on this PR. The build signature would also
have a hash of the build command besides a hash of the environment. It does not replace this PR. It also requires more work. Currently the binaries are in the testing folder, which is simply replaced during an upgrade. This procedure would need to be enhanced.

Reply:
I now set the white list to `[]`.